custom_components/tuya_local/devices/tuya_000004rime_weather_station.yaml

```diff
@@ -0,0 +1,674 @@
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 4
+            value: Cloudy and light rain
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 5
+            value: Cloudy and moderate rain
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 6
+            value: Cloudy and heavy rain
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 7
+            value: Cloudy, rain and thunder
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 8
+            value: Snowy
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 9
+            value: Cloudy and snowy
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 10
+            value: Cloudy, rainy and snowy
+            mask: 000000FF00000000000000000000000000
+      - id: 103
+        optional: true
+        type: base64
+        name: Payload
+  - entity: sensor
+    name: Local humidity
+    class: humidity
+    dps:
+      - id: 132
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: Sensor 1 temperature
+    class: temperature
+    dps:
+      - id: 133
+        optional: true
+        type: integer
+        name: sensor
+        unit: C
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: Sensor 1 humidity
+    class: humidity
+    dps:
+      - id: 134
+        optional: true
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: Sensor 2 temperature
+    class: temperature
+    dps:
+      - id: 135
+        optional: true
+        type: integer
+        name: sensor
+        unit: C
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: Sensor 2 humidity
+    class: humidity
+    dps:
+      - id: 136
+        optional: true
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: Sensor 3 temperature
+    class: temperature
+    dps:
+      - id: 137
+        optional: true
+        type: integer
+        name: sensor
+        unit: C
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: Sensor 3 humidity
+    class: humidity
+    dps:
+      - id: 138
+        optional: true
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: select
+    name: Time mode
+    icon: "mdi:clock-edit"
+    category: config
+    dps:
+      - id: 102
+        type: boolean
+        name: option
+        mapping:
+          - dps_val: true
+            value: 24h
+          - dps_val: false
+            value: 12h
+  - entity: switch
+    name: Alarm
+    icon: "mdi:alarm-light"
+    category: config
+    dps:
+      - id: 104
+        type: boolean
+        name: switch
+      - id: 110
+        type: string
+        name: Alarm state
```

Comment:
I am not sure that attribute names can contain spaces, normally words are separated by underscores, in addition to being all lower case.